source/system_stm32f4xx.c

```diff
@@ -81,8 +81,12 @@
 #include "stm32f4xx.h"
 #include "hal_tick.h"
 
-#if !defined  (HSE_VALUE) 
-  #define HSE_VALUE    ((uint32_t)8000000) /*!< Default value of the External oscillator in Hz */
+#ifndef YOTTA_CFG_HARDWARE_EXTERNALCLOCK
+#define YOTTA_CFG_HARDWARE_EXTERNALCLOCK (8000000)
+#endif
+
+#if !defined  (HSE_VALUE)
+  #define HSE_VALUE    ((uint32_t)(YOTTA_CFG_HARDWARE_EXTERNALCLOCK)) /*!< Default value of the External oscillator in Hz */
```

Comment:
So what happens if `HSE_VALUE` is defined somewhere else ? (in one of the included headers, for example)